scripts/asm-installer/install_asm

```diff
@@ -382,12 +384,19 @@ EOF
     fatal "Couldn't find key file ${KEY_FILE}."
   fi
 
-  if [[ -f "${OPERATOR_OVERLAY}" ]]; then
-    OPERATOR_OVERLAY="$(readlink -f "${OPERATOR_OVERLAY}")"
-    readonly OPERATOR_OVERLAY
-  elif [[ -n "${OPERATOR_OVERLAY}" ]]; then
-    fatal "Couldn't find yaml file ${OPERATOR_OVERLAY}."
+  while read -d ',' -r yaml_file; do
+    if [[ -f "${yaml_file}" ]]; then
```

Comment:
I guess here it should be `-e` to check if it is a file and then `readlink` will return the actual path?

Reply:
-f also checks if it's a file. -e is any file-like object, -f is an fd that's not a device or directory.

Reply:
Ok, I thought -f does not check for symbolic link. Solved.

Reply:
Just to double check, I ran these. It's true if the soft link points to a real file, and false if the link exists but points to a file that doesn't exist.

```bash
/tmp$ touch ./file
/tmp$ ln -s ./file one
/tmp$ ln -s nope two
/tmp$ [[ -f file ]]
/tmp$ echo $?
0
/tmp$ [[ -f one ]]
/tmp$ echo $?
0
/tmp$ [[ -f two ]]
/tmp$ echo $?
1
```